lib/brains/tests/test_forms.py

```diff
@@ -84,3 +85,58 @@ def test_missing_cancel_params(self):
         errors = form.errors
         # Make sure empty params are caught.
         assert 'subscription' in errors, errors.as_text()
+
+
+class TestSaleForm(BraintreeTest):
+
+    def errors(self, data):
```

Comment:
the calling code reads oddly when this is called `errors()`. Maybe just `submit()` ?